### PR TITLE
tables: single read without checking presence

### PR DESCRIPTION
**Pull request by filak-sap**
Please, consider adding this advice.

### CODE REVIEW

File: clean-abap/CleanABAP.md [hunk 1]
```diff
@@ -1063,6 +1064,32 @@ LOOP AT my_table REFERENCE INTO DATA(line).
...
+```
+
+which is more efficient than
+
+```ABAP
+IF not line_exists( my_table[ key = intup ] ).
+  RAISE EXCEPTION NEW /clean/my_data_not_found( ).
+ENDTRY.
+
+DAT(row) = my_table[ key = intup ]
```
Comment (larshp) [inline clean-abap/CleanABAP.md]: Syntax error `DAT` -> `DATA` plus add `.`

plus consider upper casing "not" to have it consistently pretty printed

Comment (jfilak) [reply clean-abap/CleanABAP.md]: Thanks, amended!

Comment (larshp) [reply clean-abap/CleanABAP.md]: plus "intup" -> "input"

Comment (jfilak) [reply clean-abap/CleanABAP.md]: :D D'oh, no comment ... amended. Thanks!

**Comment by HrFlorianHoffmann** [comment]
Did some edits:
- Shortened the main text
- Linked to related section
- Added "anti-pattern" tag
- Fixed links and typos

If you agree to these, I'll merge.

**Comment by jfilak** [comment]
Of course. Sorry, for not being able to fix all those typos :D

**Comment by HrFlorianHoffmann** [comment]
No prob. That's why we're doing reviews.

**Pull request merged**